svthalia/discord-bot
# Fix/bot status prefix

Closes #24



### Summary
Fixed issue #24

### How to test
Steps to test the changes you made:
1. Change the default prefix in the .env file
2. Run the bot and check it's status for the correct prefix.
